apps/nextjs-example/pages/index.tsx

```diff
@@ -7,8 +8,10 @@ import {
     Network,
     Provider,
     TxnBuilderTypes,
+    TypeTagParser,
```

Comment:
this is the old sdk, why we do new imports?

Reply:
Accident, removed!